codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/smithy/generators/BuilderGenerator.kt

```diff
@@ -208,7 +209,11 @@ class BuilderGenerator(
     private fun renderBuilder(writer: RustWriter) {
         writer.docs("A builder for #D.", structureSymbol)
         Attribute(derive(builderDerives)).render(writer)
+        writer.writeInline("/// This is the datatype that Builder of this module build itself into.\n")
+        writer.writeInline("pub type OutputShape = #D", structureSymbol)
```

Comment:
Prefer the Rust-specific writer fns
```suggestion
        writer.rustInline("pub type OutputShape = #D", structureSymbol)
```